src/raw_imbalances.py

```diff
@@ -121,21 +121,28 @@ def extract_actions(self, traces: list[AttributeDict], address: str) -> list[dic
                     actions.append(dict(action))
         return actions
 
-    def calculate_native_eth_imbalance(self, actions: list[dict], address: str) -> int:
```

Comment:
This change looks like a regression in terms of complexity and code style.

What was the rational?

If the change of return type is required, make only that change here, e.g. via `if in_flow == 0 and out_flow == 0: return None`.

The docstring should mention what it means if `None` is returned.